aptos-move/framework/src/built_package.rs

```diff
@@ -130,21 +122,25 @@ impl Default for BuildOptions {
             // This is false by default, because it could accidentally pull new dependencies
             // while in a test (and cause some havoc)
             skip_fetch_latest_git_deps: false,
-            bytecode_version: Some(VERSION_7),
-            compiler_version: Some(CompilerVersion::latest_stable()),
-            language_version: Some(LanguageVersion::latest_stable()),
+            bytecode_version: None,
```

Comment:
@vineethk, change back to `None` so that by default it will use compiler v2 but when `MOVE_COMPILER_V1` is set, V1 will be used.